Server/VikingVault.Services/Abstractions/IUserService.cs

```diff
@@ -9,5 +9,6 @@ public interface IUserService
     {
         User CreateUser(User user);
         User GetById(int userId);
+
```

Comment:
Remove blank lines, s'il te plait

Reply:
I don't even know this language.

But done.